### PR TITLE
Updated to document new CORS environment variables support

### PR DESCRIPTION
**Pull request by chriswk**
With the amazing contribution from @AbrahamLopez10 , the proxy now supports setting all CORS options in environment variables, simplifying docker deploys for those who need to for instance include credentials.

This PR updates the docs to document the environment variables that Abraham added in his PR.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -103,31 +103,37 @@ Keep-Alive: timeout=5
...
+| cors.preflightContinue    | `CORS_PREFLIGHT_CONTINUE`        | `false`         |    no    |                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
+| cors.optionsSuccessStatus | `CORS_OPTIONS_SUCCESS_STATUS`    | 204             |    no    | If the Options call passes CORS, which http status to return                                                                                                                                                                                                                                                                                                                                                                                                             |
```
Comment (thomasheartman) [inline README.md:136]: Super! Just one small request: could you order these alphabetically together with the rest of the "cors.<propertyName>" options?

**Review by thomasheartman — CHANGES_REQUESTED** [review]
Wonderful! Just one little nit about sort order 😇

File: README.md (hunk 1 above)
Comment (chriswk) [reply README.md]: Done in 966ca98

**Review by thomasheartman — APPROVED** [review]
Love it! Thank you :smile:

**Pull request merged**